test/e2e/launch.js

```diff
@@ -49,43 +40,64 @@ function launch(t) {
       console.log(`using cli binary`, cliBinary)
       console.log(`using node binary`, nodeBinary)
 
-      cliHome = join(testDir, `cli_home`)
-      nodeHome = join(testDir, `node_home`)
+      const cliHomePrefix = join(testDir, `cli_home`)
+      const nodeHomePrefix = join(testDir, `node_home`)
 
       fs.removeSync(`testArtifacts`)
 
       // setup first node
-      const initValues = await initLocalNode(1)
-      const nodeOneHome = nodeHome + `_1`
-      const nodeOneCliHome = cliHome + `_1`
-      console.error(`ui home: ${cliHome}`)
+      const nodeOneHome = nodeHomePrefix + `_1`
+      const nodeOneCliHome = cliHomePrefix + `_1`
+      const operatorKeyName = `testkey`
+      console.error(`ui home: ${nodeOneCliHome}`)
```

Comment:
Why are you using `console.error` here instead of `console.log`?  According to `console_error_throw.js`, writing to `console.error` signifies a failure.

Reply:
This is a typo ;)